chia/rpc/wallet_rpc_api.py

```diff
@@ -999,6 +1001,15 @@ async def send_notification(self, request) -> EndpointResult:
         await self.service.wallet_state_manager.add_pending_transaction(tx)
         return {"tx": tx.to_json_dict_convenience(self.service.config)}
 
+    async def sign_message(self, request) -> EndpointResult:
+        puzzle_hash: bytes32 = decode_puzzle_hash(request["address"])
```

Comment:
why address and not `puzzle_hash`?

Reply:
Address is more popular among the users.

Reply:
Makes sense. I just want to ensure that we have consistency (always accepting address or always accepting puzzle hash), in all the apis. It looks like some are using one and some are using the other, which is not great. I suppose its fine to use address then.